src/components/Icon/index.js

```diff
@@ -26,7 +26,7 @@ export default class Icon extends PureComponent {
   get realColor () {
     const { name, color } = this.props
     if (color !== 'custom') return color
-    return CUSTOM_ICONS.includes(name) ? name : color
+    return CUSTOM_ICONS.indexOf(name) ? name : color
```

Comment:
This is not how how to check for `indexOf`. `indexOf` returns either the position of the character within the string or `-1`. This needs to be:
```javascript
return CUSTOM_ICONS.indexOf(name) !== -1 ? name : color
```